src/views/ChargePointInstallation.vue

```diff
@@ -496,10 +496,11 @@
 								"
 							>
 								<template #help>
-									An allen Ladepunkten, denen dieses
-									Ladepunkt-Profil zugeordnet ist, werden die
-									hier eingetragenen ID-Tags zur Freischaltung
-									verwendet.
+									Wenn hier Tags eingetragen werden, können
+									nur die eingetragenen Tags zur
+									Fahrzeug-Zuordnung genutzt werden. Sind
+									keine Tags eingetragen, wird nur die
+									Zuordnung zum Fahrzeug geprüft.
```

Comment:
Ich musste den Absatz mehrfach lesen, bis ich ihn verstanden habe. Vielleicht wird es mit diesem Zusatz etwas deutlicher.
```suggestion
									Wenn hier Tags eingetragen werden, können
									nur die eingetragenen Tags zur
									Fahrzeug-Zuordnung genutzt werden. Sind
									keine Tags eingetragen, wird nur die
									Zuordnung zum Fahrzeug geprüft. In diesem
									Fall können alle Fahrzeuge diesen
									Ladepunkt nutzen.
```